Node.js/GettingStarted/processcsv.py

```diff
@@ -0,0 +1,100 @@
+import csv
+import json
+from azure.storage.blob import BlockBlobService
+import datetime
+import os
+import argparse
+
+#### TODO: Get the configuration
+
+storage_acc_name = '<storage account name>'
+storage_acc_key = '<storage account key>'
+block_blob_service = BlockBlobService(account_name=storage_acc_name,account_key=storage_acc_key)
+####
+
+def getfilename(name):
+
+    names = str(name).split('/')
+    return names[len(names)-1]
+
+
+
+def processcsvfile(fname,seperator,outdir,outfname):    
+    header = []    
+    with open(fname) as inpfile:
+        
+        allrows = csv.reader(inpfile,delimiter=seperator)
+        print("loaded file " + fname)
+        all_vals = []
+
+        for rows in allrows:
+            if isHeaderLoaded is False:
+                header = rows
+            else:
+                if len(header) > 0:
+                    i = 0
+                    line = "{"
+                    
+                    for r in rows:
+                        if i == 0:
```

Comment:
Are blank lines after `if` and `else` idiomatic Python?  They look odd to my C# eyes...